### PR TITLE
#166047734 Api endpoint for verifying clients status 

### PR DESCRIPTION
**Pull request by femitj**
**What does this PR do?**

Marks client as verified

**Description of the task to be completed**

 - Ensure the following endpoints are working as they should

   - PATCH/api/v1/users/<:user-mail>/verify

**How should this be tested manually**

 - After you clone the repo,
 - cd into it and run npm install to install node modules.
 - Make sure you have nodejs and express installed.
 - Afterwards, run npm start to initiate the app.
 - test endpoint above using postman with localhost:8080 as the host

**What are the relevant pivotal stories**
#166047734

### CODE REVIEW

File: server/middleware/auth-validations.js [hunk 1]
```diff
@@ -130,16 +131,67 @@ const validations = {
...
+    req.data = {
+      email,
+      firstname,
+      lastname,
+      address,
+      status,
+    };
+    return next();
+  },
+
```
Comment (chuzksy-codeactive) [inline server/middleware/auth-validations.js]: remove this empty line

**Review by chuzksy-codeactive — APPROVED** [review]
Looks good to me. 👍

**Review by chuzksy-codeactive — CHANGES_REQUESTED** [review]
Resolve your merge conflicts

File: server/middleware/auth-validations.js (hunk 1 above)
Comment (femitj) [reply server/middleware/auth-validations.js]: removed, 
merge conflict - resolved

**Pull request merged**